### PR TITLE
Feature/select-and-drop-transformers

### PR DESCRIPTION
**Pull request by AndreasLindhardt**

### CODE REVIEW

File: src/atc/etl/extended_transformer.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+                    if datasetKey in self.dataset_input_key_list
+                }
+            )
+        elif len(inputs) == 1:
+            df = self.process(next(iter(inputs.values())))
+        else:
+            df = self.process_many(inputs)
+
+        inputs[self.dataset_output_key] = df
+        return inputs
```
Comment (LauJohansson) [inline src/atc/etl/extended_transformer.py:49]: I would really like to have a unit-test of these logics, if this extended transformer is going to be used as a core class to ETL processes. There is some logic (if-else statements), and I think it is neccesary to tests that it behaves as expected. You should might consult with @JeppeBlixen

**Review by LauJohansson — CHANGES_REQUESTED** [review]

File: src/atc/etl/extended_transformer.py (hunk 1 above)
Comment (LauJohansson) [reply src/atc/etl/extended_transformer.py:49]: Also, if this is a kind of core transformer class, it should also get some documentation on what it is, and how to use it.

Comment (LauJohansson) [reply src/atc/etl/extended_transformer.py:49]: I have consulted with Jeppe, and @MartinBoge is going to implement this incl. with tests. I would recommend that you wait for that PR, merge it into this PR, and then remove the "ExtendedTransformer" and let DropColumnsTransformer and SelectColumnsTransformer inherit Martins transformer.

**Comment by mrmasterplan** [comment]
what does the extended consumer do that the standard comsumer does not do? Please explain. I currently cannot see the need to add this core class.

**Comment by JeppeBlixen** [comment]
@mrmasterplan and @LauJohansson. This pr was just to add out generic drop and select transformers. Could we move this discussion about the ExtendedTransformer to this pr: https://github.com/atc-net/atc-dataplatform/pull/149 Martin is working on this part.

**Comment by LauJohansson** [comment]
> @mrmasterplan and @LauJohansson. This pr was just to add out generic drop and select transformers. Could we move this discussion about the ExtendedTransformer to this pr: #149 Martin is working on this part.

Yes!

**Comment by LauJohansson** [comment]
I have changed the name of the PR to a more suitable name.

**Pull request merged**